packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualRows.ts

```diff
@@ -220,23 +222,33 @@ export const useGridVirtualRows = (apiRef: GridApiRef): void => {
         }
       }
 
-      let needScroll = !isColVisible;
+      let isRowIndexAbove = false;
+      let isRowIndexBelow = false;
 
       if (params.rowIndex || params.rowIndex === 0) {
         const elementIndex = !options.pagination
           ? params.rowIndex
           : params.rowIndex - paginationState.page * paginationState.pageSize;
 
         const currentRowPage = elementIndex / gridState.containerSizes!.viewportPageSize;
-        const scrollPosition = currentRowPage * gridState.containerSizes!.renderingZoneScrollHeight;
-
-        if (windowRef.current!.scrollTop !== scrollPosition) {
-          logger.debug(`Setting top to ${scrollCoordinates.top}`);
-          scrollCoordinates.top = scrollPosition;
-          needScroll = true;
+        const viewportHeight = gridState.containerSizes!.renderingZoneScrollHeight;
```

Comment:
it's a misleading var name I think

Reply:
True, I changed to `height`. Probably I'll revisit this hook to fix the jump, then we can discuss that again.